src/modules/App/App.js

```diff
@@ -4,16 +4,18 @@ import AppContext from "./AppContext";
 import { appReducer, initialState } from "../common/appReducer";
 import "./App.css";
 import MissionControl from "../views/MissionControl";
+import StyleSample from "../views/StyleSample";
+import Button from '../../ui/button/Button'
 
 function App() {
   const [state, dispatch] = useReducer(appReducer, initialState);
+  const [show, setShow] = useState(true)
 
   return (
     <AppContext.Provider value={[state, dispatch]}>
       <div className="App">
-        <h1 className="App-logo">🕵️</h1>
-        <p>Welcome to KidDo</p>
-        <MissionControl />
+        <Button onClick={() => setShow(!show)}>{show ? "Hide Style Guide" : "Show Style Guide"}</Button>
```

Comment:
This is such a great idea to create a reference for styles we can easily access.